linkml/generators/docgen.py

```diff
@@ -121,7 +124,7 @@ def serialize(self, directory: str = None) -> None:
         for sn, s in sv.all_slots().items():
             if self._is_external(s):
                 continue
-            n = self.name(s)
+            n = self.name(s, use_slot_uris=self.use_slot_uris)
```

Comment:
the logic here is odd. use_slot_uris is already an instance attribute on the Generator